docs/edc/EDC_BPDM_Usage.postman_collection.json

```diff
@@ -1,454 +1,6302 @@
 {
   "info": {
-    "_postman_id": "67fa306a-a4c1-4d70-80bb-370491d8d0a9",
-    "name": "EDC BPDM Usage",
+    "_postman_id": "478bfefa-0ab2-4299-993e-1800562484f5",
+    "name": "EDC-BPDM Automated Test Template",
```

Comment:
All looks good, just one comment as above for name value.
Also, I have imported collection and tried testing the same but OfferID for Generic endpoints are not being generated so couldn't go into further the details but test scripts looks good to me.

We have automation folder in repo with file /main/docs/postman/automated-test/EDC-BPDM%20Automated%20Test.postman_collection.json either we need to remove that and keep this or vice versa  with updated changes for Generic endpoints will also be fine .

Reply:
I tested it again and fixed some issue with the setup. Now it should work on our EDC